src/qt/signverifymessagedialog.cpp

```diff
@@ -30,6 +30,9 @@ SignVerifyMessageDialog::SignVerifyMessageDialog(QWidget* parent) :
     pageButtons.addButton(ui->btnVerifyMessage, pageButtons.buttons().size());
     connect(&pageButtons, SIGNAL(buttonClicked(int)), this, SLOT(showPage(int)));
 #if QT_VERSION >= 0x040700
+    ui->messageIn_VM->setPlaceholderText(tr("Enter a message to be verified"));
+    ui->signatureIn_VM->setPlaceholderText(tr("Enter a signature for the message to be verified"));
```

Comment:
53ad7cb0b7eefe01e2b4683334e0c4ad44ff0d39: `*_VM` lines were added below `*_SM` ones in the original PR. We should probably do it the same way here to avoid merge conflicts later when we are going to merge master back into develop.

Reply:
Squashed it into the commit